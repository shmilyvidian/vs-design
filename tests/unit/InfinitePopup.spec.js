import {
  shallowMount,
} from '@vue/test-utils'
import InfinitePopup from '@/packages/infinite-popup/src/index.vue'

test('render svg when image is network', () => {
  const testText = 'test'
  const wrapper = shallowMount(InfinitePopup)

  expect(wrapper).toMatchSnapshot()
});
