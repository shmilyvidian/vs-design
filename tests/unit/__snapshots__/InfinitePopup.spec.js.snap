// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render popup 1`] = `
<div class="infinite-popup">
  <popup-stub overlay="true" closeonclickoverlay="true" zindex="100" lockscroll="true" lazyrender="true" duration="300" closeicon="cross" closeiconposition="top-right" position="center">内容</popup-stub>
</div>
`;
